src/components/autosuggest/autosuggest.ui.js

```diff
@@ -43,6 +43,7 @@ export default class AutosuggestUI {
     this.input = context.querySelector(`.${baseClass}-input`);
     this.resultsContainer = context.querySelector(`.${baseClass}-results`);
     this.listbox = this.resultsContainer.querySelector(`.${baseClass}-listbox`);
+    this.resultsTitleContainer = this.resultsContainer.querySelector(`.ons-autosuggest__results-title`);
```

Comment:
This should also use `${baseClass}` for consistency

```suggestion
    this.resultsTitleContainer = this.resultsContainer.querySelector(`.${baseClass}__results-title`);
```